scripts/verify-install.sh

```diff
@@ -132,20 +137,20 @@ for signature in $SIGNATURES; do
   echo ""
 
   # Then make sure that the hash of the installed binaries can be found in the
-  # signed list of hashes.
-  if ! grep -q "$LND_SUM" "$signature"; then
-    echo "ERROR: Hash $LND_SUM for lnd not found in $signature: "
-    cat "$signature"
+  # manifest.
+  if ! grep -q "$LND_SUM" "$MANIFEST"; then
+    echo "ERROR: Hash $LND_SUM for lnd not found in $MANIFEST: "
+    cat "$MANIFEST"
     exit 1
   fi
 
-  if ! grep -q "$LNCLI_SUM" "$signature"; then
-    echo "ERROR: Hash $LNCLI_SUM for lncli not found in $signature: "
-    cat "$signature"
+  if ! grep -q "$LNCLI_SUM" "$MANIFEST"; then
+    echo "ERROR: Hash $LNCLI_SUM for lncli not found in $MANIFEST: "
+    cat "$MANIFEST"
     exit 1
   fi
 
-  echo "Verified lnd and lncli hashes against $signature"
+  echo "Verified lnd and lncli hashes against $MANIFEST"
```

Comment:
nit: I think the hashes only need to be verified once now, e.g.:
```
for sig in signatures:
    verify sig
#everyone agrees on manifest
check lnd hash
check lncli hash
```

Reply:
Agreed! Though if the hashes don't match we don't need to look at the signatures. So I put the hash check first.

Reply:
As an optimization, that's true. This is more about what surfacing what is actually wrong with the verification.

If we check hashes first, you don't know whether the manifest is actually agreed upon by all developers. If all the developers signed the correct manifest, but the manifest on github is invalid the user will see that the hash wasn't included. They might think that they've built the wrong binary, but the error that user _should_ see is that the signatures disagree with the github manifest.

Once we know everyone agrees, then we can return whether the hash of your binary is not in the signed manifest.

Signature disagreement represents a faulty/backdoored release process, the latter just means you don't have the correct binary.

Reply:
Yes, the hash check was still in the loop over each signature in the version you initially reviewed.

I now moved it to after the signature check. It totally makes sense with the scenario you describe, I just didn't think of this and hence the premature optimization :sweat_smile: 

I also updated some of the error messages to reflect your suggestion.